app/components/error-wrapper.js

```diff
@@ -1,10 +1,31 @@
 import Ember from 'ember';
 
+/**
+  `error-wrapper` handles http response status code errors and other server
+  level errors and displays an error page to the user.
+
+  ## default usage
+
+  ```Handlebars
+  {{error-wrapper model=model}}
+  ```
+
+  @class error-wrapper
+  @module Component
+  @extends Ember.Component
+ */
+
 export default Ember.Component.extend({
   classNames: ['error-wrapper', 'center-pseudo'],
 
   background: Ember.inject.service(),
 
+  /**
+    Returns a message based on the type of error thrown.
+
+    @property errorClass
+    @type string
```

Comment:
I think we should use capitalized types `@type string` --> `@type String`.
We might want to add these things to the documentation style guide.